eventbridge-pipes-sqs-to-redshift/example-pattern.json

```diff
@@ -0,0 +1,68 @@
+{
+  "title": "Step Functions to Athena",
+  "description": "How to send messages from Amazon SQS queue to Amazon Redshift Clsuter using Amazon EventBridge Pipes ",
+  "language": "Python",
+  "level": "200",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "The solution will create two Amazon SQS queues, Amazon Redshift Cluster, and an EventBridge Pipes to connect it all. EventBridge pipes consumer for messages source SQS queue",
+      "then transform the SQS message to create insert queries from data within message and insert data to Amazon Redshift cluster and add the SQS messages"
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/eventbridge-pipes-sqs-to-redshift",
+      "templateURL": "serverless-patterns/eventbridge-pipes-sqs-to-redshift",
+      "projectFolder": "eventbridge-pipes-sqs-to-redshift",
+      "templateFile": "eventbridge-pipes-sqs-to-redshift/amazon_sqs_integration_with_redshift.py"
```

Comment:
Do not include projectFolder on templateFile.
Reference: https://github.com/aws-samples/serverless-patterns/blob/main/_pattern-model/example-pattern.json#L15-L21

Reply:
Yes updated